exercises/concept/calculator-conundrum/.meta/design.md

```diff
@@ -18,3 +18,18 @@
 
 - `conditionals-if`: know how to do conditional logic.
 - `switch-statement`: know how to work with a `switch` statement.
+
+## Analyzer
+
+This exercise could benefit from the following rules in the [analyzer]:
+
+- `essential`: Verify that the solution is using the try catch statement for the division
+- `essential`: Verify that the solution is throwing an exception if the parameter "operation" is null
+- `essential`: Verify that the solution is throwing an exception if the parameter "operation" is empty
+- `essential`: Verify that the solution is throwing an exception if the parameter "operation" is not a mathematical operation.
+- `actionable`: If the solution is wrapping all the code in a try catch statement, instruct the student to only use the try catch for the division statement
+- `actionable`: If the solution uses only `if` statement, instruct the student that he/she can use the `switch case` statement also.
```

Comment:
I think it might be worth mentioning where the `switch case` could be used. For example "... can use the `switch case` statement to perform the operation".